### PR TITLE
Fix docstrings issues for torch.utils.tensorboard files and others

### LINKED ISSUE #112637: Fix docstring errors in summary.py, _caffe2_graph.py, writer.py, _pytorch_graph.py, constants.py, _utils.py, _proto_graph.py
Please fix the following issues. To test locally, run: 
 ```
pydocstyle path-to-file --count
```
Run this command before and after fixing the errors. This command prints out the number of errors at the end of the output. Paste the output to your PR description with the number before and after. 
- **File**: `torch/utils/tensorboard/_proto_graph.py`, **Entity**: `attr_value_proto`, **Line**: 8, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_proto_graph.py`, **Entity**: `attr_value_proto`, **Line**: 8, **Description**: First line should end with a period (not 'g')
- **File**: `torch/utils/tensorboard/_proto_graph.py`, **Entity**: `attr_value_proto`, **Line**: 8, **Description**: First line should be in imperative mood (perhaps 'Create', not 'Creates')
- **File**: `torch/utils/tensorboard/_proto_graph.py`, **Entity**: `tensor_shape_proto`, **Line**: 23, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_proto_graph.py`, **Entity**: `tensor_shape_proto`, **Line**: 23, **Description**: First line should end with a period (not 'g')
- **File**: `torch/utils/tensorboard/_proto_graph.py`, **Entity**: `tensor_shape_proto`, **Line**: 23, **Description**: First line should be in imperative mood (perhaps 'Create', not 'Creates')
- **File**: `torch/utils/tensorboard/_proto_graph.py`, **Entity**: `node_proto`, **Line**: 38, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_proto_graph.py`, **Entity**: `node_proto`, **Line**: 38, **Description**: First line should end with a period (not 'g')
- **File**: `torch/utils/tensorboard/_proto_graph.py`, **Entity**: `node_proto`, **Line**: 38, **Description**: First line should be in imperative mood (perhaps 'Create', not 'Creates')
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `GraphPy`, **Line**: 122, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `GraphPy`, **Line**: 122, **Description**: First line should end with a period (not 'n')
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `to_proto`, **Line**: 216, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `to_proto`, **Line**: 216, **Description**: First line should end with a period (not 'd')
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `to_proto`, **Line**: 216, **Description**: First line should be in imperative mood (perhaps 'Convert', not 'Converts')
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `parse`, **Line**: 237, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `parse`, **Line**: 237, **Description**: First line should end with a period (not 's')
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `parse`, **Line**: 237, **Description**: First line should be in imperative mood; try rephrasing (found 'This')
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `graph`, **Line**: 322, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `graph`, **Line**: 322, **Description**: First line should end with a period (not 'o')
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `graph`, **Line**: 322, **Description**: First line should be in imperative mood; try rephrasing (found 'This')
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `_set_model_to_eval`, **Line**: 368, **Description**: First line should be in imperative mood; try rephrasing (found 'A')
- **File**: `torch/utils/tensorboard/_pytorch_graph.py`, **Entity**: `_node_get`, **Line**: 385, **Description**: First line should be in imperative mood (perhaps 'Get', not 'Gets')
- **File**: `torch/utils/tensorboard/_utils.py`, **Entity**: `_prepare_video`, **Line**: 40, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_utils.py`, **Entity**: `_prepare_video`, **Line**: 40, **Description**: First line should end with a period (not ']')
- **File**: `torch/utils/tensorboard/_utils.py`, **Entity**: `_prepare_video`, **Line**: 40, **Description**: First line should be in imperative mood (perhaps 'Convert', not 'Converts')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `hparams`, **Line**: 76, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `hparams`, **Line**: 76, **Description**: First line should be in imperative mood (perhaps 'Output', not 'Outputs')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `scalar`, **Line**: 265, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `scalar`, **Line**: 265, **Description**: First line should be in imperative mood (perhaps 'Output', not 'Outputs')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `histogram_raw`, **Line**: 308, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `histogram_raw`, **Line**: 308, **Description**: First line should be in imperative mood (perhaps 'Output', not 'Outputs')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `histogram`, **Line**: 340, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `histogram`, **Line**: 340, **Description**: First line should be in imperative mood (perhaps 'Output', not 'Outputs')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `image`, **Line**: 415, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `image`, **Line**: 415, **Description**: First line should be in imperative mood (perhaps 'Output', not 'Outputs')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `image_boxes`, **Line**: 453, **Description**: First line should be in imperative mood (perhaps 'Output', not 'Outputs')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `make_image`, **Line**: 482, **Description**: First line should end with a period (not 'f')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `_get_tensor_summary`, **Line**: 734, **Description**: First line should be in imperative mood (perhaps 'Create', not 'Creates')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `_get_json_config`, **Line**: 791, **Description**: First line should be in imperative mood (perhaps 'Parse', not 'Parses')
- **File**: `torch/utils/tensorboard/summary.py`, **Entity**: `mesh`, **Line**: 802, **Description**: First line should be in imperative mood (perhaps 'Output', not 'Outputs')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: ``, **Line**: 1, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: ``, **Line**: 1, **Description**: Multi-line docstring closing quotes should be on a separate line
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: ``, **Line**: 1, **Description**: First line should end with a period (not 'e')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `__init__`, **Line**: 55, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `__init__`, **Line**: 55, **Description**: First line should be in imperative mood (perhaps 'Create', not 'Creates')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `get_logdir`, **Line**: 81, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_event`, **Line**: 85, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_event`, **Line**: 85, **Description**: First line should be in imperative mood (perhaps 'Add', not 'Adds')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_summary`, **Line**: 101, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_summary`, **Line**: 101, **Description**: First line should be in imperative mood (perhaps 'Add', not 'Adds')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_graph`, **Line**: 116, **Description**: First line should be in imperative mood (perhaps 'Add', not 'Adds')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_onnx_graph`, **Line**: 135, **Description**: First line should be in imperative mood (perhaps 'Add', not 'Adds')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `flush`, **Line**: 146, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `close`, **Line**: 153, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `reopen`, **Line**: 159, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `SummaryWriter`, **Line**: 168, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `SummaryWriter`, **Line**: 168, **Description**: First line should end with a period (not 'e')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `__init__`, **Line**: 187, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `__init__`, **Line**: 187, **Description**: First line should end with a period (not 's')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `__init__`, **Line**: 187, **Description**: First line should be in imperative mood (perhaps 'Create', not 'Creates')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `_check_caffe2_blob`, **Line**: 260, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `_check_caffe2_blob`, **Line**: 260, **Description**: First line should end with a period (not 'f')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `_get_file_writer`, **Line**: 275, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `get_logdir`, **Line**: 296, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_scalars`, **Line**: 394, **Description**: First line should be in imperative mood (perhaps 'Add', not 'Adds')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_histogram_raw`, **Line**: 501, **Description**: First line should be in imperative mood (perhaps 'Add', not 'Adds')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_pr_curve`, **Line**: 981, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_pr_curve`, **Line**: 981, **Description**: First line should be in imperative mood (perhaps 'Add', not 'Adds')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_pr_curve_raw`, **Line**: 1033, **Description**: First line should be in imperative mood (perhaps 'Add', not 'Adds')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_custom_scalars_multilinechart`, **Line**: 1069, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_custom_scalars_multilinechart`, **Line**: 1069, **Description**: First line should end with a period (not 't')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_custom_scalars_marginchart`, **Line**: 1088, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_custom_scalars_marginchart`, **Line**: 1088, **Description**: First line should end with a period (not 't')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_custom_scalars`, **Line**: 1106, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_custom_scalars`, **Line**: 1106, **Description**: First line should end with a period (not 'e')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_mesh`, **Line**: 1137, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `add_mesh`, **Line**: 1137, **Description**: First line should end with a period (not ',')
- **File**: `torch/utils/tensorboard/writer.py`, **Entity**: `flush`, **Line**: 1193, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_rename_tensorflow_style`, **Line**: 42, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_convert_to_ssa`, **Line**: 82, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_add_gradient_scope`, **Line**: 217, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_add_gradient_scope`, **Line**: 217, **Description**: First line should end with a period (not 'a')
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_replace_colons`, **Line**: 243, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_replace_colons`, **Line**: 243, **Description**: First line should end with a period (not 's')
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_fill_missing_operator_names`, **Line**: 267, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_add_tf_shape`, **Line**: 325, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_add_tf_shape`, **Line**: 325, **Description**: First line should end with a period (not 'f')
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_add_tf_shape`, **Line**: 325, **Description**: First line should be in imperative mood (perhaps 'Convert', not 'Converts')
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_set_tf_attr`, **Line**: 345, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_set_tf_attr`, **Line**: 345, **Description**: First line should end with a period (not ',')
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_operator_to_node`, **Line**: 390, **Description**: First line should be in imperative mood (perhaps 'Convert', not 'Converts')
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_blob_to_node`, **Line**: 479, **Description**: First line should be in imperative mood (perhaps 'Convert', not 'Converts')
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_clear_debug_info`, **Line**: 514, **Description**: First line should be in imperative mood (perhaps 'Remove', not 'Removes')
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_check_if_forward`, **Line**: 537, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `_operators_to_graph_def`, **Line**: 638, **Description**: First line should be in imperative mood; try rephrasing (found 'Main')
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `model_to_graph_def`, **Line**: 762, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/tensorboard/_caffe2_graph.py`, **Entity**: `model_to_graph_def`, **Line**: 762, **Description**: First line should end with a period (not 's')
- **File**: `torch/utils/hipify/constants.py`, **Entity**: ``, **Line**: 1, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/utils/hipify/constants.py`, **Entity**: ``, **Line**: 1, **Description**: No whitespaces allowed surrounding docstring text

cc @carljparker

### PR DESCRIPTION
**Pull request by ChanBong**
Fixes #112637 

Fixed pydocstyle errors. Also added a few missing docstrings for functions

Previous error count mentioned in #112637 
Current error count 

```
pydocstyle summary.py --count
0
```

```
pydocstyle _caffe2_graph.py --count
0
```

```
pydocstyle writer.py --count
0
```

```
pydocstyle _pytorch_graph.py --count
0
```

```
pydocstyle constants.py --count
0
```

```
pydocstyle _utils.py --count
0
```

```
pydocstyle _proto_graph.py --count
15
```

cc @svekars @carljparker

### CODE REVIEW

**Comment by svekars** [comment]
@ChanBong - can you please remove the missing docstrings and only fix the errors listed in the issue? Thanks!

File: torch/utils/tensorboard/_pytorch_graph.py [hunk 1]
```diff
@@ -119,8 +119,8 @@ def __init__(self, node_cpp):
 
 
 class GraphPy:
-    """Helper class to convert torch.nn.Module to GraphDef proto and visualization
-    with TensorBoard.
+    """
+    Helper class to convert torch.nn.Module to GraphDef proto and visualization with TensorBoard.
```
Comment (svekars) [inline torch/utils/tensorboard/_pytorch_graph.py:123]:
```suggestion
    A helper class to convert torch.nn.Module to GraphDef proto and visualization with TensorBoard.
```

File: torch/utils/tensorboard/_pytorch_graph.py [hunk 2]
```diff
@@ -213,10 +212,7 @@ def populate_namespace_from_OP_to_IO(self):
                 ]
 
     def to_proto(self):
-        """
-        Converts graph representation of GraphPy object to TensorBoard
-        required format.
-        """
+        """Convert graph representation of GraphPy object to TensorBoard required format."""
```
Comment (svekars) [inline torch/utils/tensorboard/_pytorch_graph.py:215]:
```suggestion
        """Convert graph representation of a GraphPy object to TensorBoard required format."""
```

File: torch/utils/tensorboard/_utils.py [hunk 3]
```diff
@@ -67,6 +71,16 @@ def is_power2(num):
 
 
 def make_grid(I, ncols=8):
+    """
```
Comment (svekars) [inline torch/utils/tensorboard/_utils.py:74]: Please remove newly added docstrings and only focus on the errors in the issue.

File: torch/utils/tensorboard/summary.py [hunk 4]
```diff
@@ -589,7 +603,19 @@ def image_boxes(
 
 
 def draw_boxes(disp_image, boxes, labels=None):
-    # xyxy format
+    """
```
Comment (svekars) [inline torch/utils/tensorboard/summary.py:606]: Please remove newly added docstrings and only focus on the errors in the issue.

File: torch/utils/tensorboard/summary.py [hunk 5]
```diff
@@ -635,6 +661,18 @@ def make_image(tensor, rescale=1, rois=None, labels=None):
 
 
 def video(tag, tensor, fps=4):
+    """
```
Comment (svekars) [inline torch/utils/tensorboard/summary.py:664]: Please remove newly added docstrings and only focus on the errors in the issue.

File: torch/utils/tensorboard/summary.py [hunk 6]
```diff
@@ -646,6 +684,16 @@ def video(tag, tensor, fps=4):
 
 
 def make_video(tensor, fps):
+    """
```
Comment (svekars) [inline torch/utils/tensorboard/summary.py:687]: Please remove newly added docstrings and only focus on the errors in the issue.

File: torch/utils/tensorboard/summary.py [hunk 7]
```diff
@@ -689,6 +737,18 @@ def make_video(tensor, fps):
 
 
 def audio(tag, tensor, sample_rate=44100):
+    """
```
Comment (svekars) [inline torch/utils/tensorboard/summary.py:740]: Please remove newly added docstrings and only focus on the errors in the issue.

File: torch/utils/tensorboard/summary.py [hunk 8]
```diff
@@ -719,6 +779,16 @@ def audio(tag, tensor, sample_rate=44100):
 
 
 def custom_scalars(layout):
+    """
```
Comment (svekars) [inline torch/utils/tensorboard/summary.py:782]: Please remove newly added docstrings and only focus on the errors in the issue.

File: torch/utils/tensorboard/summary.py [hunk 9]
```diff
@@ -756,6 +826,18 @@ def custom_scalars(layout):
 
 
 def text(tag, text):
+    """
```
Comment (svekars) [inline torch/utils/tensorboard/summary.py:829]: Please remove newly added docstrings and only focus on the errors in the issue.

File: torch/utils/tensorboard/summary.py [hunk 10]
```diff
@@ -773,6 +855,25 @@ def text(tag, text):
 def pr_curve_raw(
     tag, tp, fp, tn, fn, precision, recall, num_thresholds=127, weights=None
 ):
+    """
```
Comment (svekars) [inline torch/utils/tensorboard/summary.py:858]: Please remove newly added docstrings and only focus on the errors in the issue.

File: torch/utils/tensorboard/summary.py [hunk 11]
```diff
@@ -797,6 +898,20 @@ def pr_curve_raw(
 
 
 def pr_curve(tag, labels, predictions, num_thresholds=127, weights=None):
+    """
```
Comment (svekars) [inline torch/utils/tensorboard/summary.py:901]: Please remove newly added docstrings and only focus on the errors in the issue.

File: torch/utils/tensorboard/summary.py [hunk 12]
```diff
@@ -824,6 +939,19 @@ def pr_curve(tag, labels, predictions, num_thresholds=127, weights=None):
 
 # https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/pr_curve/summary.py
 def compute_curve(labels, predictions, num_thresholds=None, weights=None):
+    """
```
Comment (svekars) [inline torch/utils/tensorboard/summary.py:942]: Please remove newly added docstrings and only focus on the errors in the issue.

File: torch/utils/tensorboard/writer.py [hunk 13]
```diff
@@ -483,7 +503,8 @@ def add_histogram(
         walltime=None,
         max_bins=None,
     ):
-        """Add histogram to summary.
+        """
+        Add histogram to summary.
```
Comment (svekars) [inline torch/utils/tensorboard/writer.py:507]:
```suggestion
        Add a histogram to summary.
```

File: torch/utils/tensorboard/writer.py [hunk 14]
```diff
@@ -534,7 +555,8 @@ def add_histogram_raw(
         global_step=None,
         walltime=None,
     ):
-        """Adds histogram with raw data.
+        """
+        Add histogram with raw data.
```
Comment (svekars) [inline torch/utils/tensorboard/writer.py:559]:
```suggestion
        Add a histogram with raw data.
```

File: torch/utils/tensorboard/writer.py [hunk 15]
```diff
@@ -766,7 +791,8 @@ def add_figure(
         close: bool = True,
         walltime: Optional[float] = None
     ) -> None:
-        """Render matplotlib figure into an image and add it to summary.
+        """
+        Render matplotlib figure into an image and add it to summary.
```
Comment (svekars) [inline torch/utils/tensorboard/writer.py:795]:
```suggestion
        Render a matplotlib figure into an image and add it to summary.
```

File: torch/utils/tensorboard/writer.py [hunk 16]
```diff
@@ -1022,7 +1054,9 @@ def add_pr_curve(
         weights=None,
         walltime=None,
     ):
-        """Adds precision recall curve.
+        """
+        Add precision recall curve.
```
Comment (svekars) [inline torch/utils/tensorboard/writer.py:1058]:
```suggestion
        Add a precision recall curve.
```

File: torch/utils/tensorboard/writer.py [hunk 17]
```diff
@@ -1074,7 +1108,8 @@ def add_pr_curve_raw(
         weights=None,
         walltime=None,
     ):
-        """Adds precision recall curve with raw data.
+        """
+        Add precision recall curve with raw data.
```
Comment (svekars) [inline torch/utils/tensorboard/writer.py:1112]:
```suggestion
        Add a precision recall curve with raw data.
```

File: torch/utils/tensorboard/writer.py [hunk 18]
```diff
@@ -1147,8 +1188,11 @@ def add_custom_scalars_marginchart(
         self._get_file_writer().add_summary(custom_scalars(layout))
 
     def add_custom_scalars(self, layout):
-        """Create special chart by collecting charts tags in 'scalars'. Note that this function can only be called once
-        for each SummaryWriter() object. Because it only provides metadata to tensorboard, the function can be called
+        """
+        Create special chart by collecting charts tags in 'scalars'.
```
Comment (svekars) [inline torch/utils/tensorboard/writer.py:1192]:
```suggestion
        Create a special chart by collecting charts tags in 'scalars'.
```

File: torch/utils/tensorboard/writer.py [hunk 19]
```diff
@@ -1244,6 +1292,7 @@ def flush(self):
             writer.flush()
 
     def close(self):
+        """Close the event file and release resources."""
```
Comment (svekars) [inline torch/utils/tensorboard/writer.py:1295]: Please remove newly added docstrings and only focus on the errors in the issue.

File: torch/utils/tensorboard/writer.py [hunk 20]
```diff
@@ -1252,7 +1301,9 @@ def close(self):
         self.file_writer = self.all_writers = None
 
     def __enter__(self):
+        """No-op for compatibility with Python's with syntax."""
```
Comment (svekars) [inline torch/utils/tensorboard/writer.py:1304]: Please remove newly added docstrings and only focus on the errors in the issue.

File: torch/utils/tensorboard/writer.py [hunk 21]
```diff
@@ -1252,7 +1301,9 @@ def close(self):
...
         return self
 
     def __exit__(self, exc_type, exc_val, exc_tb):
+        """Flush the data and close the file on `with` exit."""
```
Comment (svekars) [inline torch/utils/tensorboard/writer.py:1308]: Please remove newly added docstrings and only focus on the errors in the issue.

**Review by svekars — COMMENTED** [review]
Pease don't fix the 'Missing docstring' errors and only fix the errors in the issue. Thank you!

**Comment by ChanBong** [comment]
I have opened a new PR #113336  for the same issue, with only fixing the error mentioned in the issue. 

I am so sorry to not notify here and you had to review this again :(.  

I think its best to close this PR.

**Pull request closed without merging**